test/Object/getOwnPropertyDescriptors.js

```diff
@@ -153,6 +153,24 @@ var tests = [
 
             verifyObjectDescriptors(desc, allTrueSymbol, allFalseSymbol);
         }
+    },
+    {
+        name:"For any property, if getOwnPropertyDescriptor(property) is undefined, that property should not be present on the result.",
+        body: function() {
+            // Adapted from: https://github.com/ljharb/test262/blob/c2eaa30b08fb1e041b7297e415b6bad8461f50dc/test/built-ins/Object/getOwnPropertyDescriptors/proxy-undefined-descriptor.js
+            var proxyHandler = {
+              getOwnPropertyDescriptor: function () {},
```

Comment:
This bug is only truly testable if `ownKeys` returns a key, that `getOwnPropertyDescriptor` returns undefined for - so it's critical to define `ownKeys` to `return [key]`.